[HUB]: Fix weekly submodule update

```
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
Error: Process completed with exit code 1.
```

https://github.com/yuri-norwood/dotfiles/runs/3082152744?check_suite_focus=true#step:4:13

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -33,7 +33,7 @@ jobs:
           git config --global user.email "Yuri Norwood"
           git config --global user.name "76186754+yuri-norwood@users.noreply.github.com"
           git submodule foreach --recursive git pull origin master
-          git commit -am "Updated submodules"
+          git commit -am "Updated submodules" && echo "Nothing to commit"
 
       # Create new branch and push to remote
       - name: Push branch

